### DISCUSSION TITLE
ASIO issues

### DISCUSSION THREAD

**Discussion by jeremunga** [jamulussoftware/jamulus#1357] [Technical Support & Help]
We are a guitar quartet just starting to use Jamulus. We can all join a server easily enough but one of the four is inaudible. He can hear us but we can't hear him. He downloaded and (we think) installed asio4all version 2.14 but it doesn't seem to have kicked in. He is on Windows 8. I have looked at the tutorial window and read the asio4all FAQ's and Jamulus troubleshooting page but haven't found the solution. Any advice?

**Comment by gilgongo** [comment]
If it's ASIO4all it sounds like he needs may not have hit the right device for output in the ASIO setup window perhaps? I think there's an "advanced" view of it that makes this clearer.

**Reply by dcorson-ticino-com** [reply]
You could also try the version ASIO4ALL 2.15beta.
For some people this has solved problems.

**Reply by jeremunga** [reply]
Thank you I will try it
